Components/GameBoard/GameBoard.jsx

```diff
@@ -287,6 +288,116 @@ export class GameBoard extends React.Component {
         this.setState(newState);
     }
 
+    renderBoard(thisPlayer, otherPlayer) {
+        if(this.props.rookeryDeck) {
+            return (
+                <RookerySetup
+                    cards={ this.props.cards }
+                    cardSize={ this.props.user.settings.cardSize }
+                    deck={ this.props.rookeryDeck }
+                    onCardMouseOut={ this.onMouseOut }
+                    onCardMouseOver={ this.onMouseOver }
+                    onSubmit={ this.props.submitRookeryPrompt }
+                    packs={ this.props.packs }
+                    players={ Object.values(this.props.currentGame.players) }
+                    restrictedList={ this.props.restrictedList } />);
+        }
+
+        return [
```

Comment:
Huh.  Interesting way of doing it, I'd have just wrapped them in a div, unless there's layout reasons that that doesn't work.  But if this works then fine